pop-api/src/v0/fungibles/tests.rs

```diff
@@ -0,0 +1,142 @@
+use super::{FungiblesError, PSP22Error};
```

Comment:
This tests module feels unnecessary, can we move the tests to the relevant module